src/main/webapp/WEB-INF/tags/sys/innerWireTransfer.tag

```diff
@@ -26,6 +26,7 @@
 <c:set var="wireTransAttributes" value="${DataDictionary.PaymentSourceWireTransfer.attributes}" />
 <%-- Cornell Customization --%>
 <c:set var="wireTransExtendedAttributes" value="${DataDictionary.PaymentSourceWireTransferExtendedAttribute.attributes}" />
+<c:set var="maskNonProd" value="${kfunc:isNonProductionEnvAndUnmaskingTurnedOff()? 'true' : 'false'}" />

```

Comment:
You may want to add a space before the question mark.

Also, I would suggest adding appropriate customization comments to the areas you modified, especially the part that changes the "mask" derivation logic.

Reply:
Thank you Chad, adding changes